src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSType.cs

```diff
@@ -127,6 +130,8 @@ public sealed class Function<T> : JSType where T : JSType
         /// <summary>
         /// Marshal as JavaScript <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function">Function</see> type.
         /// </summary>
+        /// <typeparam name="T1">Type of marshaled parameter or result</typeparam>
```

Comment:
Suggestion applies to all similar texts in the PR:
```suggestion
        /// <typeparam name="T1">The type of marshaled parameter or result.</typeparam>
```